### ISSUE TITLE
[BUG] - max weight lower than max rep

### ISSUE THREAD

**Issue by jwallet** [madmustachecompany/Blast-Workout-App#51]
### Description

My PRs show a max weight with a lower reps count with the same weight in max rep section

### Reproduction steps

```bash
Edit a workout
```

### Screenshots

![Screenshot_20230727-082218_Blast.jpg](https://github.com/madmustachecompany/Blast-Workout-App/assets/23088305/2ea6c854-b97c-4f91-a7ca-ec124c2f63e4)

### OS

Android

**Comment by noelielc** [comment]
Good point, there seems to be a problem when exercises are performed with zero weight, such as bodyweight exercises.
